workspaces/icecap-host/Makefile

```diff
@@ -23,6 +23,7 @@ WORKSPACE_DIR = $(abspath ..)
 OUT_DIR ?= $(abspath test-collateral)
 MEASUREMENT_FILE = $(abspath ../icecap-runtime/css-icecap.bin)
 MEASUREMENT_PARAMETER = --css-file $(MEASUREMENT_FILE)
+PROFILE_FLAG = release
```

Comment:
This statement is overridden by `common.mk`. Did you mean `PROFILE = release`?

Reply:
Yes, I did.